test/pcaps/ikev2_nat_t/esp_sa

```diff
@@ -0,0 +1,7 @@
+# This file is automatically generated, DO NOT MODIFY.
```

Comment:
Do you use this? Otherwise please remove it

Reply:
Removed